autofish.lua

```diff
@@ -0,0 +1,220 @@
+-- handles automatic fishing jobs to limit the number of fish the fortress keeps on hand
+-- autofish [enable | disable] <max> [min] [--include-raw | -r]
```

Comment:
docs should go in `docs/autofish.rst`
You can copy `docs/stripcaged.rst` to get the skeleton structure. More info here: https://docs.dfhack.org/en/latest/docs/dev/Documentation.html